packages/core-mobile/app/contexts/SwapContext/SwapContext.tsx

```diff
@@ -168,11 +169,13 @@ export const SwapContextProvider = ({
         .then(([result, err]) => {
           if (err || (result && 'error' in result)) {
             setSwapStatus('Fail')
-            AnalyticsService.captureWithEncryption('SwapTransactionFailed', {
-              address: activeAccount.addressC,
-              chainId: activeNetwork.chainId
-            })
-            showTransactionErrorToast({ message: humanizeSwapErrors(err) })
+            if (!isUserRejectedError(err)) {
+              AnalyticsService.captureWithEncryption('SwapTransactionFailed', {
+                address: activeAccount.addressC,
+                chainId: activeNetwork.chainId
+              })
+              showTransactionErrorToast({ message: humanizeSwapErrors(err) })
+            }
```

Comment:
we were reporting and showing toast even for the user rejected error. this fixes it.